### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

**Comment by SATYAPAL-BISHNOI** [comment]
.

File: portable/README [hunk 1]
```diff
@@ -1 +1,2 @@
+strlcat & strlcpy source from:
 ftp://ftp.openbsd.org/pub/OpenBSD/src/lib/libc/string/
```
Comment (SATYAPAL-BISHNOI) [inline portable/README:2]: why

File: Makefile.am [hunk 2]
```diff
@@ -35,6 +35,9 @@ EXTRA_DIST = \
 	build-aux/gen-version \
 	build-aux/tap-driver.sh \
 	doc \
+	portable/inet_ntop.h \
+	portable/strlcat.h \
+	portable/strlcpy.h \
 	$(TEMPLATE_FILES) \
 	# End of EXTRA_DIST
```
Comment (SATYAPAL-BISHNOI) [inline Makefile.am:43]: this is  right way

**Review by SATYAPAL-BISHNOI — APPROVED** [review]

**Comment by remyleone** [comment]
I think we should transfer comments on this pull request: https://github.com/dun/munge/pull/142

**Pull request closed without merging**